app/Models/Order.php

```diff
@@ -31,14 +31,6 @@ class Order extends Model
         'lid_name',
         'group_id',
         'user_id',
-        'order_status_id',
-        'status',
-    ];
-
-    protected $fillable = [
```

Comment:
Ik weet niet waarom er alweer een tweede fillable is, maar beide kwamen overeen met de kolommen uit de db (hadden meer of minder). Nu klopt het, op de timestaps na.